Sources/SnapshotPreviewsCore/ExpandingViewController.swift

```diff
@@ -92,5 +104,42 @@ public final class ExpandingViewController: UIHostingController<EmergeModifierVi
     }
   }
 
+//  MARK: - Timer
+
+  func startTimer() {
+      if timer == nil {
+          startTime = Date()
+          timer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true) { [weak self] _ in
```

Comment:
Maybe this time interval should be higher to prevent it from firing a bunch of times without doing anything?